index.js

```diff
@@ -4,15 +4,46 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
+header.style.color = "red"
 
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+const profile = document.querySelector("#profile")
+const name = profile.children[1];
+name.innerHTML = traveler.name;
+const nickName = profile.children[2];
+nickName.innerHTML = traveler.nickname;
+const pfp = profile.children[0];
+pfp.src= traveler.photo
+pfp.alt= traveler.name
 
 
 /***** Deliverable 4 *****/
+const animals = document.querySelector("#animals")
+const animalSightings = traveler.animalSightings
+for(let i = 0; i <traveler.animalSightings.length; i++){
```

Comment:
Great! Since you already created the variable animalSightings, you can do 
i < animalSightings.length

Try refactoring this using a forEach. This would allow you to avoid referencing the index when setting attributes